Office365-Admin/add-users/get-access-to-and-back-up-a-former-user-s-data.md

```diff
@@ -45,11 +45,11 @@ If you remove a user's Microsoft 365 license but don't delete the account, you r
 
 To preserve a former user's OneDrive for Business documents you first access their OneDrive site and then move the files. 
 
-1. In the admin center, go to the **User management > [Active users](https://go.microsoft.com/fwlink/p/?linkid=834822)** page.
+1. In the admin center, go to the **Users** > **[Active users](https://go.microsoft.com/fwlink/p/?linkid=834822)** page.
```

Comment:
The ">" in each instance on the page needs to have the escape character "\" so that the markdown renders correctly. So each of these should be changed to "\>"